### PR TITLE
Refactor python manage.py send_emails --eds_email=n to select and email n applicants

### PR DESCRIPTION
**Pull request by hancush**
The current behavior selects chunks of applicants and sends emails to the subset on the correct step. It'd be easier to work with if it reliably emailed n applicants at a time. This PR implements that and adds a test.

### CODE REVIEW

**Comment by hancush** [comment]
TO-DO: Remedy my own misunderstanding by updating `eds_sent` on all related applications in one database transaction.

File: lots_admin/management/commands/send_emails.py [hunk 1]
```diff
@@ -45,28 +45,39 @@ def add_arguments(self, parser):
...
+
+                print('Notified {}'.format(applicant))
+
+                # Find all EDS-ready applications for for given applicant 
+                # and set `eds_sent` = True
+
+                all_applications = Application.objects\
+                                              .filter(email=app.application.email)\
+                                              .filter(applicationstatus__current_step__step=7)
+
```
Comment (fgregg) [inline lots_admin/management/commands/send_emails.py]: why should we filter on step_7?

**Comment by hancush** [comment]
This will toggle `eds_sent` on all applications tied to the email address we find in our initial selection. When the applicant submits the EDS, [this endpoint](https://github.com/datamade/large-lots/blob/master/lots_client/views.py#L424-L445) gets pinged. It _should_ filter for those on step 7, because we only want those applications to be moved to "EDS completed." 

Doesn't that leave open the possibility that some applications will be marked as "EDS sent" but will never be advanced? Say someone has applications on step 5 and 7 and completes the EDS. The one on step 7 will advance, but step 5 (assuming it makes it past the alderman letter and lottery, if applicable) will be left hanging because the applicant won't complete another EDS, which is what triggers advancement to step 8.

**Comment by hancush** [comment]
@fgregg I've made the changes we discussed.

**Pull request merged**